tests/test_processor.py

```diff
@@ -31,3 +36,199 @@ def test_repositories(self):
             self.project.repositories()
             request_mock.assert_called_once_with("1/repositories", method='get')
             mock.assert_called_once_with(repositories_hash)
+
+
+class TestRepository(TestCase):
```

Comment:
We thought about this and considered creating a superclass for testing independently the methods with date as string and as a dateutil object. This would save us some lines (around 9).

On the other hand we opted for this implementation with more lines of code because like this the bahaviour of the methods is explicit and the maintainability is better for someone to understand the test in the future.